extensions/adobe/experience/adcloud/searchaggregatedperformancebyad.schema.json

```diff
@@ -0,0 +1,85 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "https://ns.adobe.com/experience/adcloud/searchaggregatedperformancebyad",
+  "title": "Search Advertising Aggregated Performance Measures By Ad",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/time-series"],
+  "description": "Search Advertising Aggregated Performance Measures By Ad is a collection of aggregated measures pulled from Search Advertising Platforms by Ad/Keyword.",
+  "definitions": {
+    "searchaggregatedperformancebyad": {
```

Comment:
Does XDM mandate all class names need to be in lower case? Makes it easy to read if we can use camelCase